hyst.yaml

```
@@ -0,0 +1,14 @@
+hystrix:
+  command:
+    default:
+      execution:
+        isolation:
+          thread:
+            timeoutInMilliseconds: 200 
+          semaphore:
+            maxConcurrentRequests: -1 
+      circuitBreaker:
+        enabled: true
+        requestVolumeThreshold: -10 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)